src/extensionApi.ts

```diff
@@ -79,6 +81,41 @@ export class CommandHandler {
         }
     }
 
+    public async debugServer(mode: string, context?: Protocol.ServerState): Promise<Protocol.StartServerResponse> {
```

Comment:
Does mode need to be an argument here? It's only ever going to be 'debug' anyway.